Remora.Commands/Trees/CommandTreeBuilder.cs

```diff
@@ -173,6 +352,8 @@ private IEnumerable<IChildNode> ToChildNodes(IEnumerable<Type> moduleTypes, IPar
     private IEnumerable<CommandNode> GetModuleCommands(Type moduleType, IParentNode parent)
     {
         var methods = moduleType.GetMethods();
+        var isInUnnamedGroup = !moduleType.TryGetGroupName(out var gn) || gn == string.Empty;
```

Comment:
It has occurred to me that this line is likely faulty and has caused a small bug in Remora in which command nodes will subsume their parent's attributes incorrectly; considering how Remora handles attributes in most cases (read: walking up the tree), this is likely unnecessary in the first place.

I will be opening a PR to rectify this issue, as well as adding additional tests.

Reply:
![image](https://github.com/user-attachments/assets/6342f2d4-db34-4ee1-8b7c-cab646273368)